Lib/fontTools/ttLib/tables/D__e_b_g.py

```diff
@@ -0,0 +1,16 @@
+from . import DefaultTable
+import json
+
+class table_D__e_b_g(DefaultTable.DefaultTable):
+
+	def decompile(self, data, ttFont):
+		self.data = json.loads(data)
+
+	def compile(self, ttFont):
+		return str.encode(json.dumps(self.data))
```

Comment:
why `str.encode` instead of the more idomatic `s.encode()` method on a str instrance?
Also, I think we want to explicitly encode as UTF-8, I'm not sure what the default is these days but explicit is better than implicit.